### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by XhmikosR**
@bezoerb not sure what's the coverage here, I hope we don't break anything, so please review carefully each patch and let me know.

Notes:

1. please do not squash the patches
2. ~~it would be nice if we updated to the latest meow version~~ **EDIT:** I did this, but please confirm
3. remove Travis CI and AppVeyor from the repo hooks/apps/integrations
4. enable automated security fixes for the repo

CI preview: <https://github.com/XhmikosR/filter-css/runs/316392604>

PS. I went with Node.js v6.0.0 because that's what's supported in critical 1.x.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -110,23 +113,21 @@ function reduceRules(ignore, opts) {
...
+	let sheet;
 	try {
 		sheet = css.parse(read(stylesheet));
-	} catch (err) {
+	} catch (error) {
 		sheet = css.parse(stylesheet);
 	}
 
-	sheet.stylesheet.rules = _.reduce(sheet.stylesheet.rules, reduceRules(ignore, opts), []);
+	sheet.stylesheet.rules = (sheet.stylesheet.rules || []).reduce(reduceRules(ignore, opts), []);
```
Comment (XhmikosR) [inline index.js]: If this can't be undefined let me know and I'll remove the fallback

Comment (bezoerb) [reply index.js]: According to the [readme](https://github.com/reworkcss/css#stylesheet) and the [code](https://github.com/reworkcss/css/blob/master/lib/parse/index.js) this should always be an array.

If `css.parse(...)` fails it should throw an error and we won't get here so i think we could safely remove the fallback

**Review by bezoerb — APPROVED** [review]

**Comment by XhmikosR** [comment]
I also have these 2 patches:

https://github.com/bezoerb/filter-css/commit/caa69eb9c0d1f661ec20e3a610e3698a647a2427
https://github.com/bezoerb/filter-css/commit/b67199f2b7161ed970d9bacd570f2a7d520cc77d

Not 100% sure about it, but calling showHelp() already exits automatically. Also, I'm not sure why we need the setTimeout; it seems to work without it.

**Comment by XhmikosR** [comment]
@bezoerb feel free to merge and release a new version which should either be a minor release or v1.0.0.

**Comment by XhmikosR** [comment]
But please check out the TODO https://github.com/bezoerb/filter-css/pull/4#issue-344177868

**Comment by bezoerb** [comment]
3 & 4 is done

**Comment by bezoerb** [comment]
> I also have these 2 patches:
> 
> [caa69eb](https://github.com/bezoerb/filter-css/commit/caa69eb9c0d1f661ec20e3a610e3698a647a2427)
> [b67199f](https://github.com/bezoerb/filter-css/commit/b67199f2b7161ed970d9bacd570f2a7d520cc77d)
> 
> Not 100% sure about it, but calling showHelp() already exits automatically. Also, I'm not sure why we need the setTimeout; it seems to work without it.

You're right. Seems like we don't need the setTimeout. 👍

**Comment by bezoerb** [comment]
Thanks @XhmikosR

**Comment by bezoerb** [comment]
@XhmikosR: Can you create another PR for caa69eb and b67199f
I think we can go with v1.0.0 after these are merged.

**Comment by bezoerb** [comment]
Thanks again :)

**Pull request merged**